angular-app/src/app/pages/admin-panel/admin-panel/admin-panel.component.ts

```diff
@@ -56,6 +64,35 @@ export class AdminPanelComponent {
     );
   }
 
+  changeUserRole(data, user: IUser) {
+    let newUser: IUser = {
+      id: user.id,
```

Comment:
not sure if this needs to be `_id` for the backend to pick up

Reply:
Took your suggestion